bitcoin/src/blockdata/transaction.rs

```diff
@@ -612,8 +623,8 @@ impl cmp::Ord for Transaction {
 }
 
 impl Transaction {
-    /// Maximum transaction weight for Bitcoin Core 25.0.
     // https://github.com/bitcoin/bitcoin/blob/44b05bf3fef2468783dcebf651654fdd30717e7e/src/policy/policy.h#L27
+    /// Maximum transaction weight for Bitcoin Core 25.0.
```

Comment:
No need for this change IMO, I assume it was because of the code comment on `BASE_WEIGHT`.

Reply:
I think you mentioned in a previous PR that developer comments go above doc comments?

Reply:
I don't remember saying so, I don't have a policy on docs/rustdocs in my mind. You could be mixing up with docs above attributes, which I remember commenting on recently.